src/lib/spinel/spinel.h

```diff
@@ -3637,6 +3664,143 @@ enum
      */
     SPINEL_PROP_NEIGHBOR_TABLE_MULTI_RADIO_INFO = SPINEL_PROP_OPENTHREAD__BEGIN + 16,
 
+    /// SRP Client Start
+    /** Format: `b(6Sb)` - Write only
+     * Required capability: `SPINEL_CAP_SRP_CLIENT`.
+     *
+     * Writing to this property allows user to start or stop the SRP client operation with a given SRP server.
+     *
+     * Written value format is:
+     *
+     *   `b` : TRUE to start the client, FALSE to stop the client.
+     *
+     * When used to start the SRP client, the following fields should also be included:
+     *
+     *   `6` : SRP server IPv6 address (only when SRP client to be started).
+     *   `U` : SRP server port number (only when SRP client to be started).
+     *   `b` : Boolean to indicate whether or not to emit SRP client events (using `SPINEL_PROP_SRP_CLIENT_EVENT`).
```

Comment:
Is this also "only when SRP client to be started"?

Reply:
Yes. We only receive events from SRP client when it is started.